XenModel/Messages.resx

```diff
@@ -9442,6 +9442,12 @@ The VM protection policy for this VM does not have automatic archiving configure
   <data name="POOL_FAILURE_TOLERATE" xml:space="preserve">
     <value>{0} failure tolerated</value>
   </data>
+  <data name="POOL_FORBIDS_AUTOMATIC_RESTARTS" xml:space="preserve">
+    <value>Automatic post-update tasks are not supported for at least one server to be updated.</value>
+  </data>
+  <data name="POOL_FORBIDS_AUTOMATIC_UPDATES" xml:space="preserve">
+    <value>Automatic updates are not supported for this server.</value>

```

Comment:
It's just occurred to me that we have another two messages saying "Automatic update is not supported", could you please change this to singular so they're all the same.

Also, I had a rethink about the top message, I think "Post-update tasks cannot be carried out automatically for all the servers to be updated." conforms better with existing messages in this context.